shell/platform/darwin/ios/framework/Source/FlutterViewController.mm

```diff
@@ -760,12 +750,12 @@ - (void)surfaceUpdated:(BOOL)appeared {
 - (void)viewDidLoad {
   TRACE_EVENT0("flutter", "viewDidLoad");
 
-  if (_engine && _engineNeedsLaunch) {
-    [_engine.get() launchEngine:nil libraryURI:nil entrypointArgs:nil];
-    [_engine.get() setViewController:self];
-    _engineNeedsLaunch = NO;
-  } else if ([_engine.get() viewController] == self) {
-    [_engine.get() attachView];
+  if (self.engine && self.engineNeedsLaunch) {
+    [self.engine launchEngine:nil libraryURI:nil entrypointArgs:nil];
+    [self.engine setViewController:self];
+    self.engineNeedsLaunch = NO;
+  } else if (self.engine.viewController == self) {
+    [self.engine attachView];
   }
```

Comment:
(Definitely not in scope for this PR, but it sure feels like this method and the last method could use some refactoring into engine methods given that they mostly walls of `self.engine` calls...)